### PR TITLE
[[ API 26 ]] Update android compile sdk to API 26

### PR DESCRIPTION
**Pull request by livecodeali**
(cherry picked from commit 63a113adc37ec76a5484f600b4741b599725562a)

### CODE REVIEW

**Comment by livecodeian** [comment]
@livecode-vulcan review ok 845bc76

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeian ok 845bc7621779f9cb368753494ce0013e10b78e19

**Comment by livecode-vulcan** [comment]
😞 test failure 845bc7621779f9cb368753494ce0013e10b78e19

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1493)
* try-community-armv6-android-api9: [failure](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1509)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1483)
* try-community-universal-ios-iphoneos11.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.4/builds/10)
* try-community-universal-ios-iphonesimulator11.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.4/builds/10)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1487)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1513)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1493)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1507)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1492)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1502)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1542)
* try-community-x86_64-win32: [pending](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1482)

**Comment by livecodeali** [comment]
We will need to actually update the toolchain etc before merging this.

File: config.py [hunk 1]
```diff
@@ -412,18 +412,22 @@ def guess_android_build_tools(sdkdir):
...
         ndk = guess_android_tooldir('android-ndk')
         if ndk is None:
             error('Android NDK not found; set $ANDROID_NDK')
         opts['ANDROID_NDK'] = ndk
 
+    if opts['ANDROID_API_VERSION'] is None:
+        opts['ANDROID_API_VERSION'] = '26'
+     
+    if opts['ANDROID_PLATFORM'] is None:
+        opts['ANDROID_PLATFORM'] = 'android-26'
```
Comment (montegoulding) [inline config.py]: @livecodeali would it be a good idea to concatenate `ANDROID_API_VERSION` here so these can't get out of sync?

Comment (livecodeali) [reply config.py]: Heh, you mean like I've done in the env template? Yes, not sure why I didn't do that!

**Comment by livecodeali** [comment]
Thirdparty PR: https://github.com/livecode/livecode-thirdparty/pull/115

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 1841f33

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 1841f334027b575630fdcd50e73208f493e7e6cc

**Comment by livecode-vulcan** [comment]
😞 test failure 1841f334027b575630fdcd50e73208f493e7e6cc

* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1542)
* try-community-armv6-android-api26: [failure](https://vulcan.livecode.com/builders/try-community-armv6-android-api26/builds/27)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1516)
* try-community-universal-ios-iphoneos11.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.4/builds/43)
* try-community-universal-ios-iphonesimulator11.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.4/builds/43)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1546)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1540)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1535)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1574)
* try-community-x86_64-win32: [pending](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1514)

**Pull request merged**